cpp/src/io/csv/reader_impl.cu

```diff
@@ -152,8 +153,10 @@ std::vector<std::string> get_column_names(std::vector<char> const& row,
           --col_name_len;
         }
 
-        string const new_col_name(row.data() + prev, col_name_len);
-        col_names.push_back(removeQuotes(new_col_name, parse_opts.quotechar));
+        string new_col_name{
+          remove_quotes(std::string_view{row.data() + prev, static_cast<std::size_t>(col_name_len)},
+                        parse_opts.quotechar)};
+        col_names.push_back(std::move(new_col_name));
```

Comment:
Please check for style but in this case, I think we don't need the temp variable
```suggestion
        col_names.push_back(remove_quotes(std::string_view{row.data() + prev, static_cast<std::size_t>(col_name_len)},
                        parse_opts.quotechar));
```